source/release-notes/1.8.txt

```diff
@@ -56,21 +56,27 @@ Many-to-One Migrations
 
 To learn more, see :ref:`c2c-beta-many-to-one`.
 
-Set a Migration Name
-````````````````````
+Namespace Remapping
+```````````````````
+
+Starting in {+c2c-beta-program-short+}, ``mongosync`` can migrate data from one 
+database on the source cluster into a new database on the destination. This 
+allows you to change database names during migration.
```

Comment:
Is the part about migrating a database from one cluster to another needed here? It almost reads a bit like the migration is the new feature when really its the renaming part? Should it be simply just:

```
Starting in {+c2c-beta-program-short+}, you can change a source database name during a ``mongosync`` migration.
```
Or whats existing on the Change Database Name page:

```
Starting in {+c2c-beta-program-short+}, you can remap database names during sync. 
```